tests/integration/run.sh

```diff
@@ -47,7 +47,6 @@ export SAMPLES_DIR=samples
 # device address and credentials
 export DEVICE_ADDR=$TCB_DEVICE
 export DEVICE_USER="torizon"
-export DEVICE_PASS="1"
```

Comment:
Now that DEVICE_PASS was renamed to DEVICE_PASSWORD and the latter must be passed whenever TCB_DEVICE is set (right?) maybe we should have a check here to see if the user is passing both pieces of information.

Reply:
I did this check on last pushs! Thanks @rborn-tx!